rdr_service/resource/schemas/participant.py

```diff
@@ -14,6 +14,14 @@
 from rdr_service.resource.constants import SchemaID
 
 
+class SexualOrientationEnum(Enum):
```

Comment:
Where are these options derived from?  Codebook or data glossary?   Hoping to include links to relevant source documentation in our code.

Reply:
There is no prior source for this mapping for the Sexual Orientation values, I created the mapping here.